### PR TITLE
chore: Update dependencies

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
I've changed Prettier's `arrowParens` to `always` as that's [the recommended way](https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-arrowparens-to-always-7430httpsgithubcomprettierprettierpull7430-by-kachkaevhttpsgithubcomkachkaev) from now on.

### CODE REVIEW

File: .prettierrc.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ const prettier = require("kcd-scripts/prettier");
 
 module.exports = {
   ...prettier,
+  arrowParens: "always",
```
Comment (robinmetral) [inline .prettierrc.js:5]: Do we need to run the files through Prettier to follow to new rule? Or do we have no occurrence of this yet?

**Review by robinmetral — COMMENTED** [review]
✨

File: .prettierrc.js (hunk 1 above)
Comment (MichaelDeBoey) [reply .prettierrc.js:5]: I've run the files through Prettier, but they're already compliant. 🙂
We don't have functions with only 1 argument that has no default value (yet) 😉

**Pull request merged**